src/pages/de/reference/configuration-reference.md

```diff
@@ -262,6 +262,28 @@ Eine Markdown-Seite wird als Entwurf betrachtet, wenn sie die Frontmatter-Eigens
 ```
 
 
+### markdown.mode
+
+<p>
+
+**Typ:** `'md' | 'mdx'`<br>
+**Standard:** `mdx`
+</p>
```

Comment:
Do we still need the `<p>` wrapper for this? I think this should work too:

```suggestion
**Typ:** `'md' | 'mdx'`<br>
**Standard:** `mdx`
```

Reply:
OK!

Reply:
If it helps: I can only *barely* tolerate it because it's NWTWWHB!